### PR TITLE
feat(console): add impersonation tag to audit log

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

Add "impersonation" tag to the token change event in Audit Logs.

## Testing

截屏2024-07-18 下午2 22 40

截屏2024-07-18 下午2 25 56

## Checklist

- [ ] `.changeset`
- [ ] unit tests
- [ ] integration tests
- [ ] necessary TSDoc comments

### CODE REVIEW

File: packages/console/src/components/AuditLogTable/components/EventName/index.module.scss [hunk 1]
```diff
@@ -4,6 +4,7 @@
   display: flex;
   align-items: center;
   white-space: nowrap;
+  gap: 8px;
```
Comment (charIeszhao) [inline packages/console/src/components/AuditLogTable/components/EventName/index.module.scss:7]: nit
```suggestion
  gap: _.unit(2);
```

File: packages/console/src/pages/AuditLogDetails/index.module.scss [hunk 2]
```diff
@@ -14,6 +14,9 @@
     .eventName {
       color: var(--color-text);
       font: var(--font-title-1);
+      display: flex;
+      align-items: center;
+      gap: 8px;
```
Comment (charIeszhao) [inline packages/console/src/pages/AuditLogDetails/index.module.scss:19]: unit

File: packages/console/src/pages/AuditLogDetails/index.tsx [hunk 3]
```diff
@@ -84,7 +85,12 @@ function AuditLogDetails() {
           <Card className={styles.header}>
             <EventIcon isSuccess={data.payload.result === 'Success'} />
             <div className={styles.content}>
-              <div className={styles.eventName}>{logEventTitle[data.key]}</div>
+              <div className={styles.eventName}>
+                {logEventTitle[data.key]}
+                {data.key === 'ExchangeTokenBy.TokenExchange' && (
+                  <Tag status="alert">Impersonation</Tag>
```
Comment (charIeszhao) [inline packages/console/src/pages/AuditLogDetails/index.tsx:91]: i18n

Comment (wangsijie) [reply packages/console/src/pages/AuditLogDetails/index.tsx:91]: We don't support i18n in audit logs.

**Review by charIeszhao — APPROVED** [review]

**Pull request merged**